templates/server-tls-ca-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+{{ if .Values.scoop.caCert }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.scoop.caConfigMapName }}
```

Comment:
Why does this need to be configurable? Can the chart just determine it?

Reply:
just that we can define it in the same place we use it (values.yaml of the chart installation)